src/ts/BitmovinId3FramesExtractor.ts

```diff
@@ -123,11 +79,13 @@ export class BitmovinId3FramesExtractor {
     return size;
   }
 
-  private readBytesAsNumberArray(buffer: Uint8Array, noOfBytes: number): Array<number> {
-    const numberArray = new Array<number>(noOfBytes); //new Int8Array(noOfBytes);
+  private readBytesAsNumberArray(buffer: Uint8Array, noOfBytes: number): number[] {
+    const numberArray: number[] = [noOfBytes];
```

Comment:
You're actually creating a new array and adding the value of `noOfBytes` as the first entry.

```suggestion
    const numberArray: number[] = [];
```

Reply:
apologies thought [N] would create array w/ size of [N]. Updated.